health/notifications/alarm-notify.sh.in

```diff
@@ -182,9 +183,9 @@ fi
 # -----------------------------------------------------------------------------
 # defaults to allow running this script by hand
 
-[ -z "${NETDATA_USER_CONFIG_DIR}" ] && NETDATA_USER_CONFIG_DIR="@configdir_POST@"
-[ -z "${NETDATA_STOCK_CONFIG_DIR}" ] && NETDATA_STOCK_CONFIG_DIR="@libconfigdir_POST@"
-[ -z "${NETDATA_CACHE_DIR}" ] && NETDATA_CACHE_DIR="@cachedir_POST@"
+[ -z "${NETDATA_USER_CONFIG_DIR}" ] && NETDATA_USER_CONFIG_DIR="/etc/netdata"
```

Comment:
These four lines need to remain as they were. The way it works, is you run ./netdata-installer.sh from the root directory of the code, which then creates alarm_notify.sh from alarm_notify.sh.in, replacing these with the correct paths.